src/utils/utils.js

```diff
@@ -108,6 +111,8 @@ const curateFilters = [
 const types = [
   { value: 'composer', label: 'Composer', provider: 'packagist' },
   { value: 'pod', label: 'Pod', provider: 'cocoapods' },
+  { value: 'conda', label: 'Conda', provider: 'conda-forge' },
+  { value: 'condasrc', label: 'Conda Sources', provider: 'conda-forge' },
```

Comment:
Currently, the drop file feature in the workspace view (http://localhost:3000/workspace) only allows for one provider (conda-forge) per type (conda) of components.  An example of json file content can be found at https://github.com/clearlydefined/website/blob/83be6d02bcaf8d498188aedbc0c063f306b96214/src/components/__tests__/PageDefinitions.test.js#L55